tools/StaticAnalysis/Exceptions/AzureRM.DataFactoryV2/BreakingChangeIssues.csv

```diff
@@ -10,3 +10,5 @@
 "Microsoft.Azure.Commands.DataFactoryV2.dll","Microsoft.Azure.Commands.DataFactoryV2.GetAzureDataFactoryPipelineCommand","Get-AzureRmDataFactoryV2Pipeline","0","1020","The cmdlet 'Get-AzureRmDataFactoryV2Pipeline' no longer has output type 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.DataFactoryV2.Models.PSPipeline]'.","Make cmdlet 'Get-AzureRmDataFactoryV2Pipeline' return type 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.DataFactoryV2.Models.PSPipeline]'."
 "Microsoft.Azure.Commands.DataFactoryV2.dll","Microsoft.Azure.Commands.DataFactoryV2.GetAzureDataFactoryDatasetCommand","Get-AzureRmDataFactoryV2Dataset","0","1020","The cmdlet 'Get-AzureRmDataFactoryV2Dataset' no longer has output type 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.DataFactoryV2.Models.PSDataset]'.","Make cmdlet 'Get-AzureRmDataFactoryV2Dataset' return type 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.DataFactoryV2.Models.PSDataset]'."
 "Microsoft.Azure.Commands.DataFactoryV2.dll","Microsoft.Azure.Commands.DataFactoryV2.GetAzureDataFactoryIntegrationRuntimeCommand","Get-AzureRmDataFactoryV2IntegrationRuntime","0","1020","The cmdlet 'Get-AzureRmDataFactoryV2IntegrationRuntime' no longer has output type 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.DataFactoryV2.Models.PSIntegrationRuntime]'.","Make cmdlet 'Get-AzureRmDataFactoryV2IntegrationRuntime' return type 'System.Collections.Generic.List`1[Microsoft.Azure.Commands.DataFactoryV2.Models.PSIntegrationRuntime]'."
+"Microsoft.Azure.Commands.DataFactoryV2.dll","Microsoft.Azure.Commands.DataFactoryV2.GetAzureDataFactoryIntegrationRuntimeMetricCommand","Get-AzureRmDataFactoryV2IntegrationRuntimeMetric","0","3030","The generic type for 'property CpuUtilization' has been changed from 'System.Nullable`1[System.Double]' to 'System.Nullable`1[System.Int32]'. ","Change the generic type for 'property CpuUtilization' back to 'System.Nullable`1[System.Double]'."
```

Comment:
Neither of these are false positives.  For the first, please create a wrapper PS type for the SDK type where you mitigate the breaking change.  For the second, please add the alias back to the cmdlet.

Reply:
Both here are bug fixes. The first one was double in swagger by mistake, service always returned Int for it, and it is not breaking for customers since int autoconverts to double.
The second one was wrong alias - the Set-AzureRmDatafactoryV2Dataset cmdlet had an alias New-AzureRmDatafactoryV2 instead of New-AzureRmDatafactoryV2Dataset. Set-AzureRmDatafactoryV2 also had alias New-AzureRmDatafactoryV2 and whichever was last in the list was winning. I fixed it, by the build thinks it was a breaking change.